packages/spark-core/components/_accordions.scss

```diff
@@ -57,6 +57,7 @@
   max-width: 100%; // We want mobile nav items not to be capped at $accordion-max-width
 }
 
+// Accordion Navigation Styles
```

Comment:
Did you change your mind about consolidating these?

Reply:
made a flag for and deleted the line by line commenting as per our convo